app-arch/zstd/patches/zstd-1.3.5-gcc2.patchset

```diff
@@ -18,22 +18,22 @@ index 8459970..cb91678 100644
  	@echo compiling dynamic library $(LIBVER)
  ifneq (,$(filter Windows%,$(OS)))
 diff --git a/programs/Makefile b/programs/Makefile
-index 85d1db1..2de1675 100644
+index 579a349..78ef1a8 100644
 --- a/programs/Makefile
 +++ b/programs/Makefile
-@@ -39,10 +39,9 @@ CPPFLAGS+= -I$(ZSTDDIR) -I$(ZSTDDIR)/common -I$(ZSTDDIR)/compress \
+@@ -39,10 +39,10 @@ CPPFLAGS+= -I$(ZSTDDIR) -I$(ZSTDDIR)/common -I$(ZSTDDIR)/compress \
             -I$(ZSTDDIR)/dictBuilder \
             -DXXH_NAMESPACE=ZSTD_
  CFLAGS  ?= -O3
 -DEBUGFLAGS+=-Wall -Wextra -Wcast-qual -Wcast-align -Wshadow \
--            -Wstrict-aliasing=1 -Wswitch-enum -Wdeclaration-after-statement \
 +DEBUGFLAGS+=-Wall         -Wcast-qual -Wcast-align -Wshadow \
+             -Wstrict-aliasing=1 -Wswitch-enum -Wdeclaration-after-statement \
```

Comment:
Oops, you dropped the removal of these 3 options in `zstd-1.3.5-gcc2.patchset`.
gcc2 does not understand them:
```
cc1: Invalid option `-Wstrict-aliasing=1'
cc1: Invalid option `-Wswitch-enum'
cc1: Invalid option `-Wdeclaration-after-statement'
```
I'm pushing a commit which fixes this.